Jtmalarecki add map to index

jtmalarecki opened the pull request:
Added Google Map component. Added it to the index page.

jtmalarecki committed:
Added scriptjs dependency.

jtmalarecki committed:
Created Google Map component.

jtmalarecki committed:
Added env variable to store Google Maps API key.

jtmalarecki committed:
Added Google Map component to the index page.

- Tweaked the v-container to fill the page.
- Changed dark theme usage to light theme.
- Removed default logos and content from the index.

JeremiahPinto commented on components/GoogleMap.vue:
```diff
@@ -0,0 +1,44 @@
+  },
+  mounted() {
+    // TODO: A reference error will occur if scriptjs is define above and not here
+    scriptjs = require('scriptjs')
```
Remove the `let` above and declare as `const` here in mounted

JeremiahPinto commented on components/GoogleMap.vue:
```diff
@@ -0,0 +1,44 @@
+  },
+  methods: {
+    // The callback after the Google Maps JS API loads
+    initMap: () => {
```
Declaring arrow functions in methods can cause issues, just use the short hand notation i.e.
`initMap() {`

JeremiahPinto commented on layouts/default.vue:
```diff
@@ -58,7 +58,7 @@
     </v-toolbar>
     <v-content>
-      <v-container>
+      <v-container fill-height>
```
Add `class="pa-0 ma-0"` to the container to fill the page with just the map

jtmalarecki replied:
When I add class="pa-0 ma-0" the container collapses to 0 height. I've been using height=100% and width=100% for the map, but I think these take 100% of 0. I'm not sure how to get the map to fill the available area. I'm thinking with a flexbox and the grow property, but I haven't yet experimented with this.

jtmalarecki replied:
Scratch that, I was replacing fill-height. However, the v-container does not fill the width of the screen on desktop as it's the max-width property is set to 900px.

JeremiahPinto reviewed (changes requested):
Check out the single-line comments I made in the files, otherwise it was pretty good.

jtmalarecki committed:
Fixed variable declaration and arrow function.

jtmalarecki committed:
Updated the GoogeMap component.

- Removed scriptjs dependency, instead using google maps api loader.
- Made mounted() an async function.

jtmalarecki committed:
Changed container css to fill the screen with map.

jtmalarecki committed:
Merge branch 'master' into jtmalarecki-add-map-to-index
